### ISSUE TITLE
neutrino-preset-jest: eslint not considering jest during development

### ISSUE THREAD

**Issue by Dbuggerx** [mozilla-neutrino/neutrino-dev#186]
- _What version of Neutrino are you using_?
  - 5.6.0
- _Are you trying to use any presets? If so, which ones, and what versions?_
  - neutrino-preset-react 5.6.0
  - neutrino-preset-airbnb-base 5.6.0
  - neutrino-preset-jest 5.6.0
- _Are you using the Yarn client or the npm client? What version?_
  - yarn 0.20.3
- What version of Node.js are you using?
  - 7.5.0
- What operating system are you using?
  - GNU Linux (Fedora 25)
- What did you do?
  - Installed _neutrino-preset-jest_ 
- What did you expect to happen?
  - Eslint config ([defined in .eslintrc](https://neutrino.js.org/presets/neutrino-preset-airbnb-base/#eslintrc-config)) to include jest _plugin_ and _env_
- What actually happened, contrary to your expectations?
  - While using _VSCode_, I see lint errors (related to jest) while editing my tests

# My temporary solution:
Adding
```javascript
api.config.module
  .rule('lint')
  .use('eslint')
  .tap(options => merge(options, {
    plugins: ['jest'],
    envs: ['jest']
  }));
```
to my [.eslintrc](https://neutrino.js.org/presets/neutrino-preset-airbnb-base/#eslintrc-config) 

# What I believe would fix the issue:
Move [this block](https://github.com/mozilla-neutrino/neutrino-dev/blob/master/packages/neutrino-preset-jest/src/index.js#L63-L67) to [line 37](https://github.com/mozilla-neutrino/neutrino-dev/blob/master/packages/neutrino-preset-jest/src/index.js#L37). Because the way it's currently implemented, it's only good during test execution.

**Comment by constgen** [comment]
Your investigation helped a lot. Fixed in https://github.com/mozilla-neutrino/neutrino-dev/pull/212

**Comment by eliperelman** [comment]
Resolved in #212.
